plugins/ipam/dhcp/dhcp_test.go

```diff
@@ -118,6 +134,10 @@ var _ = Describe("DHCP Operations", func() {
 	BeforeEach(func() {
 		dhcpServerStopCh = make(chan bool)
 
+		tmpDir, err = getTmpDir()
```

Comment:
We should balance this in AfterEach() with a os.RemoveAll(tmpDir) to clean things up for each test.

Reply:
I fixed this in last push which ensured that /run/cni existed.    (I was able to reproduce test "hang" when /run didn't contain cni directory)